GitUI/Interops/DTBGOPTS.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Runtime.InteropServices;
+
+namespace System
+{
+    internal static partial class NativeMethods
+    {
+        [StructLayout(LayoutKind.Sequential)]
+        public struct DTBGOPTS
+        {
+            public int dwSize;
+            public int dwFlags;
+            public RECT rcClip;
+        }
```

Comment:
https://docs.microsoft.com/en-us/dotnet/standard/native-interop/best-practices#common-windows-data-types

```suggestion
        [StructLayout(LayoutKind.Sequential)]
        public struct DTBGOPTS
        {
            public uint dwSize;
            public uint dwFlags;
            public RECT rcClip;
        }
```